frontend/package.json

```diff
@@ -33,20 +33,26 @@
     "@testing-library/jest-dom": "4.2.4",
     "@testing-library/react": "9.3.3",
     "@testing-library/user-event": "7.1.2",
+    "@types/enzyme": "3.10.5",
+    "@types/enzyme-adapter-react-16": "1.0.6",
```

Comment:
Adding missing enzyme dependencies

Reply:
Woops, forgot to include that on my last PR. Thanks!